index.bs

```diff
@@ -1966,6 +1966,40 @@ See also:
 
 * <a href="https://www.w3.org/2001/tag/doc/promises-guide">Writing Promise-Using Specifications</a>
 
+<h4 id="sync-callbacks">When to use callback functions instead</h4>
+
+<p>If an algorithm requires that callback functions respond
+synchronously (e.g. to take or veto some action), then that is a
```

Comment:
```suggestion
synchronously (e.g., to take or veto some action), then that is a
```

I'm not sure about our conventions here; this is the one that I've been taught to use. 

... checks ...

damn.  Well.